src/core_plugins/kibana/public/management/app.html

```diff
@@ -14,9 +14,11 @@
 
       <!-- Tabs. -->
       <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
-        <span ng-if="!sectionName" class="kuiLocalTab">
-          {{::section.display}}
-        </span>
+        <header ng-if="!sectionName" id="tabHeader" tabindex="0">
+          <h2 class="kuiLocalTab">
+            {{::section.display}}
+          </h2>
+        </header>
```

Comment:
Minor nit, but I think the original issue had a slight error in that it referred to a "header", but really meant to refer to a "heading". So I think this markup should become this:

```html
        <h2 class="kuiLocalTab" ng-if="!sectionName" id="tabHeader" tabindex="0">
          {{::section.display}}
        </h2>
```